src/index.ts

```diff
@@ -249,9 +253,10 @@ export const write = (logInstance: Log): void => {
  * If enabled, calls log function.
  * @param {Logger} logger
  * @param {String} namespace
+ * @param {boolean} canForceWrite
  * @return {Logger}
  */
-export const syncLogger = (logger: Logger, namespace: string): Logger => {
+export const syncLogger = (logger: Logger, namespace: string, canForceWrite?: boolean): Logger => {
```

Comment:
I also don't think we add the parameter here.

Reply:
same syncLogger add function logMethod. CreateLogger forward this param for add a logMethod in all level.